engine/gamesys/src/gamesys/scripts/script_tilemap.cpp

```diff
@@ -263,10 +264,25 @@ namespace dmGameSystem
             assert(top + 1 == lua_gettop(L));
             return 1;
         }
-
-        bool flip_h = lua_toboolean(L, 6);
-        bool flip_v = lua_toboolean(L, 7);
-        SetTileGridTile(component, layer_index, cell_x, cell_y, tile, flip_h, flip_v);
+        bool flip_h = 0;
+        bool flip_v = 0;
+        bool rotated90 = 0;
+        if (lua_isnumber(L, 6))
```

Comment:
Is this enough?
What if developers used 0/1 instead of true/false ?

Reply:
I added one more condition